index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Darren Cedrone's Portfolio </title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+    <header>
+        <div class="name">Darren Cedrone</div>
+        <div class="link1"> <a href="https://www.linkedin.com/in/darren-cedrone-164409137">LinkedIn</a></div>
+        <div class="link2"> <a href="https://github.com/darrencedrone">GitHub</a></div>
```

Comment:
The convention on the web is that links within the site use the same tab but links off to another site throw a new tab